boards/arm/bl5340_dvk/bl5340_dvk_cpuapp_common.dts

```diff
@@ -65,6 +65,14 @@
 		};
 	};
 
+	gpio_fwd: nrf-gpio-forwarder {
+		compatible = "nordic,nrf-gpio-forwarder";
```

Comment:
Is there anything that needs to be done to enable this (i.e. Kconfig) or does happen automatically if the network core is enabled?

Reply:
Should happen automatically when network core is enabled (`CONFIG_BOARD_ENABLE_CPUNET=y`).
For referenc: https://github.com/zephyrproject-rtos/zephyr/commit/81c48c3e684f1277b89e711b72366a5a676bc880